src/collective/cover/content.py

```diff
@@ -148,6 +150,40 @@ def get_referenced_objects(self):
                 refs |= getObjectsFromLinks(self, links)
         return refs
 
+    def delete_tile(self, tile_id):
+        """Delete tile
+
+           To delete tile we just need the ID, the type can be anything.
+        """
+        tile = self.restrictedTraverse('foo/' + tile_id)
```

Comment:
@datakurre when I try to instantiate a deleted tile (to delete it's data) I get this exception:

```python
ipdb> self.restrictedTraverse(u'foo/' + tile_id)
*** NotFound: c
ipdb> self.restrictedTraverse(u'collective.cover.base/' + tile_id)
*** NotFound: c
ipdb> self.restrictedTraverse(u'collective.cover.basic/' + tile_id)
*** NotFound: c
```

Can you help me please understand why this code don't work (while it is almost the same code from `get_tile` method?